WORKSPACE

```diff
@@ -11,6 +11,14 @@ http_archive(
     ],
 )
 
+http_archive(
```

Comment:
I am sorry, I am not sure what this is for. Apologies.

Reply:
Without this, the Bazel build command to generate protos fails because zlib isn't found. See https://github.com/bazelbuild/bazel/issues/15136. I'll update the PR summary.

Reply:
Thank you for the explanation. TIL.